### PR TITLE
chore: send schema

### PR DESCRIPTION
**Pull request by GuillaumeCisco**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: django_forest/utils/apimap_errors.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+APIMAP_ERRORS = {
+    0: {
```
Comment (guillaumedeslandes) [inline django_forest/utils/apimap_errors.py:2]: `0` normally means "no error". You're sure about this value?

File: django_forest/utils/forest_api_requester.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    def error_msg(url):
+        return f'Cannot reach Forest API at {url}, it seems to be down right now.'
+
+    @staticmethod
+    def forest_api_url():
+        return getattr(settings, 'FOREST', {}).get('FOREST_URL', os.getenv('FOREST_URL', 'https://api.forestadmin.com'))
+
+    @classmethod
+    def _get_url(cls, route):
+        if route.startswith('https://'):
```
Comment (guillaumedeslandes) [inline django_forest/utils/forest_api_requester.py:31]: What if route starts by `http://`?

File: django_forest/utils/to_bool.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    't', 'T',
+    'y', 'Y', 'yes', 'YES',
+    'true', 'True', 'TRUE',
+    'on', 'On', 'ON',
+    '1', 1,
+    True
+}
+
+
+def to_bool(value):
```
Comment (guillaumedeslandes) [inline django_forest/utils/to_bool.py:11]: You can use [distutils.util.strtobool](https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool).
Just put a try/catch as it throws with "invalid" values.

Comment (guillaumedeslandes) [reply django_forest/utils/to_bool.py:11]: Tested with python 2.7 and 3.8.

File: django_forest/utils/apimap_errors.py (hunk 1 above)
Comment (GuillaumeCisco) [reply django_forest/utils/apimap_errors.py:2]: This is something I have seen in the `forest-express` liana: https://github.com/ForestAdmin/forest-express/blob/master/src/services/apimap-sender.js#L15

`forest-rails` does not manage it this way. I will review this part for making it more pythonic thanks to `requests`

File: django_forest/utils/forest_api_requester.py (hunk 2 above)
Comment (GuillaumeCisco) [reply django_forest/utils/forest_api_requester.py:31]: I don't know... This is something weird I have seen in `forest-rails` : https://github.com/ForestAdmin/forest-rails/blob/master/app/services/forest_liana/forest_api_requester.rb#L19
I don't find a similar thing in `forest-express`...

File: django_forest/utils/to_bool.py (hunk 3 above)
Comment (GuillaumeCisco) [reply django_forest/utils/to_bool.py:11]: right! that's great!
thanks!

**Review by guillaumedeslandes — APPROVED** [review]

**Pull request merged**